solr/core/src/test/org/apache/solr/search/facet/TestJsonFacets.java

```diff
@@ -3226,6 +3269,26 @@ private static String str(AVLTreeDigest digest) {
     return sb.toString();
   }
 
+  private static boolean doesFacetMethodSupportMinCountZero() {
+    return FacetField.FacetMethod.DEFAULT_METHOD != FacetField.FacetMethod.DVSTRING;
+  }
+
+  private static boolean doesFacetMethodSupportNumericFields() {
+    return FacetField.FacetMethod.DEFAULT_METHOD != FacetField.FacetMethod.DVSTRING;
+  }
+
+  private static boolean doesFacetMethodSupportTermsPrefix() {
+    return FacetField.FacetMethod.DEFAULT_METHOD != FacetField.FacetMethod.DVSTRING;
+  }
+
+  private static boolean doesFacetMethodSupportSortedSetDocValues() {
+    return FacetField.FacetMethod.DEFAULT_METHOD != FacetField.FacetMethod.DVSTRING;
+  }
+
+  private static boolean doesFacetMethodSupportFieldsWithoutDocValues() {
```

Comment:
I'm curious how these tests work for other facet methods like dvhash? And other methods that don't support all params like the above checks say

Reply:
I _think_ they build an uninverted index from the terms in that case?